src/components/CHtml/index.meta.js

```diff
@@ -1,15 +1,62 @@
+import { binding } from '@nsoft/chameleon-sdk/src/utility';
```

Comment:
Please import this into utility file and then import utility here like `@/utility` so that we import from npm `@nsoft/chameleon-sdk` in fewer places. There is this import in several places.